### DISCUSSION TITLE
ExternalError: TypeError: e.indexOf is not a function at app/tabulator/Tabulator/__init__.py:116

### DISCUSSION THREAD

**Discussion by mg3146** [anvilistas/tabulator#61] [Q&A]
Hi,

I have been using the Tabulator without issue, but this morning it's started throwing the following error:

```
ExternalError: TypeError: e.indexOf is not a function
at app/tabulator/Tabulator/__init__.py:116
```

I'm not sure where its coming from, as the traceback is pretty limited...  Any suggestions or hints are appreciated!

**Comment by s-cork** [comment]
What are you setting the data to be?

**Reply by mg3146** [reply]
A dictionary that has the same keys as Tabulator columns, but basically just blank string values.     (Its for the case where my program returns zero results to display)

**Reply by s-cork** [reply]
the data property should be a list of dictionaries so maybe that's the issue

**Reply by mg3146** [reply]
I'll give it a shot, thx!
